TheMET/TheMET/ViewControllers/AboutAppViewController.swift

```diff
@@ -0,0 +1,137 @@
+            self.tableView.topAnchor.constraint(equalTo: self.leftVersionLabel.bottomAnchor,constant: 15),
+            self.tableView.heightAnchor.constraint(equalToConstant: 130 + 53.67)
+        ])
+        NSLayoutConstraint.activate([
+            self.copyRightLabel.leadingAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.leadingAnchor, constant: 20),
+            self.copyRightLabel.trailingAnchor.constraint(equalTo: self.view.trailingAnchor),
+            self.copyRightLabel.topAnchor.constraint(equalTo: self.tableView.bottomAnchor, constant: 15)
+        ])
+        
+        self.tableView.rowHeight = UITableView.automaticDimension
+        self.tableView.register(AboutAppViewCell.self, forCellReuseIdentifier: AboutAppViewCell.aboutAppViewCellIdentifier)
+        self.tableView.dataSource = self
+        self.tableView.delegate = self
+    }
+    
+    private func addLabel(_ label : UILabel,color: UIColor?, title: String, textAlignment: NSTextAlignment) {
+        label.backgroundColor = .clear
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.apply(font: NSLocalizedString("serif_font", comment: ""), color: color, fontSize: 16, title: title)
+        label.textAlignment = textAlignment
+        self.view.addSubview(label)
+    }
+    
+    private func getAppVersion() -> String {
+        if let version = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String {
+            return version
+        } else {
+            return ""
+        }
+    }
+    
+    func showSafariPage(urlString: String) {
+        if let url = URL(string: urlString) {
+            let configuration = SFSafariViewController.Configuration()
+            configuration.entersReaderIfAvailable = true
+            let safariViewController = SFSafariViewController(url: url, configuration: configuration)
+            present(safariViewController, animated: true)
+        } else {
+            print ("error")
+        }
+    }
+    
+    //  UITableViewDelegate, UITableViewDataSource
+    
+   func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+       if let cell = tableView.dequeueReusableCell(withIdentifier: AboutAppViewCell.aboutAppViewCellIdentifier, for: indexPath) as? AboutAppViewCell {
+           cell.backgroundColor = .clear
+           cell.selectionStyle = .none
+           if indexPath.row == 0 {
+               cell.label.apply(font: NSLocalizedString("serif_font", comment: ""), color: UIColor(named: "plum"), alignment: .left, fontSize: 16, title: NSLocalizedString("privacy_policy_label", comment: ""))
+           } else {
+               cell.label.apply(font: NSLocalizedString("serif_font", comment: ""), color: UIColor(named: "plum"),alignment: .left, fontSize: 16, title: NSLocalizedString("the_use_of_met_api_label", comment: ""))
+           }
+           return cell
+       } else {
+           print ("Error")
```

Comment:
The same. Try to put some details of the error into the print. It is useless without a context,